### DISCUSSION TITLE
Images are uploaded in base64 encoding

### DISCUSSION THREAD

**Discussion by codingmachine16** [GrapesJS/grapesjs#3893] [Q&A]
Hi @artf 

When we use upload images using Grapejs. It stores images in base64 encoding then the page size exceeds from 16MB and mongodb does not allow the page to store in the document. 

RangeError [ERR_OUT_OF_RANGE]: The value of "offset" is out of range. It must be >= 0 && <= 17825792. 

What is the solution in grapesjs for uploading images as image link rather than encoding text?

Please help in this regard. Thanks

**Comment by artf** [comment]
You can [configure the asset uploader](https://grapesjs.com/docs/modules/Assets.html#uploading-assets)
